README.md

```diff
@@ -108,9 +108,9 @@ This app will output a number of files, representing the outputs of each of the
 
 Parliament2 is available as an app on DNAnexus at https://platform.dnanexus.com/app/parliament2 (note: a DNAnexus account is required to access this link; you can create one at https://platform.dnanexus.com/login). The documentation for the app is included both on the DNAnexus platform and in the `dx_app_code` directory of this repository. A DNAnexus account is required to access the platform.
 
-To run Parliament2 on DNAnexus, your input BAM file must be already on the DNAnexus platform. To run Parliament2 using the graphic interface, simply click the "Run" button from the app page and select your inputs. To run Parliament2 using the command-line interface, run the command `dx run parliament2 -h` and follow the guide generated. For more information on running executables on DNAnexus, see the [guide to running apps and applets](https://wiki.dnanexus.com/Command-Line-Client/Running-Apps-and-Applets).
+To run Parliament2 on DNAnexus, your input BAM file must be already on the DNAnexus platform. To run Parliament2 using the graphic interface, simply click the "Run" button from the app page and select your inputs. To run Parliament2 using the command-line interface, run the command `dx run parliament2 -h` and follow the guide generated. For more information on running executables on DNAnexus, see the [guide to running apps and applets](https://documentation.dnanexus.com/user/running-apps-and-workflows/running-apps-and-applets).
```

Comment:
* `-h` generates just an overview of the help. Just doing a `dx run` produces the run wizard, which I view as more of a guide.

```suggestion
To run Parliament2 on DNAnexus, your input BAM file must be already on the DNAnexus platform. To run Parliament2 using the DNAnexus platform web interface, click the "Run" button from the app page and select your inputs. To run Parliament2 using the command-line interface, run the command `dx run parliament2` and follow the guide generated. For more information on running executables on DNAnexus, see the [guide to running apps and applets](https://documentation.dnanexus.com/user/running-apps-and-workflows/running-apps-and-applets).
```